samples/cubes/convertStatusToString.js

```diff
@@ -0,0 +1,96 @@
+module.exports = (status) => {
```

Comment:
Nice! In future it may be good to extract EnumDSSCubeStates into a separate file, but this is still very useful for demonstration purposes.